### DISCUSSION TITLE
trying to access historical and active state procurement acquisitions

### DISCUSSION THREAD

**Discussion by joaquin329** [OP-TED/eForms-SDK#1029] [Q&A] [answered]
Dear colleagues,

I hope this message finds you well. My team and I have been exploring the public procurement data available through the TED platform, particularly focusing on accessing the historical and active acquisition data from different European Union member states. However, despite our efforts to utilize the provided APIs and documentation, we’ve encountered some challenges in successfully retrieving the necessary data, particularly through SPARQL queries.

We would greatly appreciate your guidance on how to best navigate these APIs, especially regarding any technical specifications or common issues that might prevent successful data extraction. Additionally, if there are specific endpoints, query examples, or any prerequisites we should be aware of, your advice would be invaluable.

Thank you for your time and assistance. We look forward to your response and any resources or instructions you could share to help us resolve these issues.

Kind regards,

**Comment by rouschr** [comment]
Dear Joaquin,

here we can help you with questions about the eForms SDK, you may occasionally get help about related topics but this seems to be about the TED API, I think you should contact their support: 
https://european-union.europa.eu/contact-eu/write-us_en

In order to get a meaningful response it would help to more precise about what exactly you tried and what failed (your SPARQL query for example).

**Comment by ferrakaop** [comment, marked as answer]
Here are the options for reusing TED data: https://ted.europa.eu/en/simap/developers-corner-for-reusers
